examples/serial/README.md

```diff
@@ -48,110 +48,70 @@ zig build -Dsdk=/path/to/sdk -Dboard=qemu_virt_aarch64 qemu
-their own colour. Upon initialisation, the transmit virtualiser will print the name of each client
-in the colour assigned to it.
-3. **SERIAL_SWITCH_CHAR** and **SERIAL_TERMINATE_NUM** - these characters control the receive
-virtualisers input switching mechanism. To switch the input stream to a different client, input
-**SERIAL_SWITCH_CHAR** followed by up to 4 numeric characters corresponding to the new client
-number, and terminate numeric input with **SERIAL_TERMINATE_NUM**. Upon success there will be no
-output, while upon error the receive virtualiser will print a debug failure message. Client 0
-receives input upon initialisation.
-4. **UART_DEFAULT_BAUD** - this determines the baud rate that the uart driver will configure for
-the device. Baud rate is always set explicitly instead of detected automatically.
-5. **SERIAL_CONSOLE_BEGIN_STRING** - this string is printed by the transmit virtualiser upon
-initialisation completion. This is to support input beginning in the interfacing serial server.
-
-If the system file is changed, or the serial subsystem is included into another system, this config
-file will need to be edited or re-created to reflect the new system. Be sure to check that the 
-`*_init\_sys` functions correctly initialise each protection domains data structures.
-
-## Interfacing with Other Systems
-To include the serial subsystem into an existing system, the following steps must be taken:
-* **.system File**
-You must update your system file to include serial data and queue regions for each client and the
-uart driver. You must also include the uart driver, transmit virtualiser, and optionally the
-receive virtualiser protection domains. Finally you must include channels between your clients and
-the virtualisers, as well as between the virtualisers and the uart driver.
-* **`serial_config` File**
-A new `serial_config` file must be created for your system, containing relevent details of the
-system file including client names and queue sizes, as well as updated initialisation functions
-for clients and virtualisers.
-* **Makefile**
-You must include directories for **SERIAL_COMPONENTS**, the **UART_DRIVER** and your
-**SERIAL_CONFIG_INCLUDE**. You must also supply **SERIAL_NUM_CLIENTS**. You must add the uart
-driver, transmit virtualiser and optionally the receive virtualiser to your image list. You must
-add your serial include directory to your cflags, and finally you must include the uart driver
-and serial_components make files. For each component you wish to have access to the serial
-subsystem, you must link their printf object file with `libsddf_util.a` as opposed to
-`libsddf_util_debug.a`. This will ensure printf invokes the serial _sddf_putchar.
-* **Protection Domains**
-Each protection domain that outputs to serial must include the serial queue library as well as
-`serial_config.h`. They must also have the following declarations/definitions:
+## Description
```

Comment:
In this section I think we should remove 'serial server' just since it adds confusion since we call them clients already.